spec/components/schemas.yaml

```diff
@@ -644,11 +626,11 @@ components:
         bbox:
           $ref: "./schemas.yaml#/components/schemas/bbox"
         geometry:
+          $ref: "./schemas.yaml#/components/schemas/itemGeometry"
           # we could use the 'original' schema, but it doesn't
           # contain a useful example, hence we use our own
-          # schema with appropriate example
-          # $ref: https://geojson.org/schema/Geometry.json
-          $ref: "./schemas.yaml#/components/schemas/geometryGeoJSON"
+          # schema with appropriate example. Original schemas
+          # were defined here: https://geojson.org/schema/Geometry.json
```

Comment:
Comment should be before the reference or totally removed, actually for such case the comment in git message is more appropriate and would always be up to date.

Reply:
I moved it before the reference and shortened it